build/run-backup.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+set -e
+
+request_data() {
+	case "${STORAGE_TYPE}" in
+		"s3")
+			cat <<-EOF
+				{
+				    "destination": "${BACKUP_DEST}",
+				    "storage": {
+				        "type": "${STORAGE_TYPE}",
+				        "verifyTLS": ${VERIFY_TLS},
+				        "s3": {
+				            "bucket": "${S3_BUCKET}",
+				            "endpointUrl": "${AWS_ENDPOINT}",
+				            "accessKey": "${AWS_ACCESS_KEY_ID}",
+				            "secretKey": "${AWS_SECRET_ACCESS_KEY}",
+				            "region": "${AWS_DEFAULT_REGION}",
+				            "storageClass": "${S3_STORAGE_CLASS}"
+				        }
+				    }
+				}
+			EOF
+			;;
+		"gcs")
+			cat <<-EOF
+				{
+				    "destination": "${BACKUP_DEST}",
+				    "storage": {
+				        "type": "${STORAGE_TYPE}",
+				        "verifyTLS": ${VERIFY_TLS},
+				        "gcs": {
+				            "bucket": "${GCS_BUCKET}",
+				            "endpointUrl": "${GCS_ENDPOINT}",
+				            "accessKey": "${ACCESS_KEY_ID}",
+				            "secretKey": "${SECRET_ACCESS_KEY}",
+				            "storageClass": "${GCS_STORAGE_CLASS}"
+				        }
+				    }
+				}
+			EOF
+			;;
+		"azure")
+			cat <<-EOF
+				{
+				    "destination": "${BACKUP_DEST}",
+				    "storage": {
+				        "type": "${STORAGE_TYPE}",
+				        "verifyTLS": ${VERIFY_TLS},
+				        "azure": {
+				            "containerName": "${AZURE_CONTAINER_NAME}",
+				            "storageAccount": "${AZURE_STORAGE_ACCOUNT}",
+				            "accessKey": "${AZURE_ACCESS_KEY}",
+				            "endpointUrl": "${AZURE_ENDPOINT}",
+				            "storageClass": "${AZURE_STORAGE_CLASS}"
+				        }
+				    }
+				}
+			EOF
+			;;
+	esac
+}
+
+request_backup() {
+	local http_code=$(
+		curl \
+			-d "$(request_data)" \
+			-H "Content-Type: application/json" \
+			-w httpcode=%{http_code} \
+			"http://${SRC_NODE}:6033/backup/${BACKUP_NAME}" 2>/dev/null \
+			| sed -e 's/.*\httpcode=//'
+	)
```

Comment:
```suggestion
	local http_code=$(
		curl \
			-d "$(request_data)" \
			-H "Content-Type: application/json" \
			-w httpcode=%{http_code} \
			-o /dev/null \
			-s \
			"http://${SRC_NODE}:6033/backup/${BACKUP_NAME}"
	)
```

We should redirect output to `/dev/null` because on error we will receive a `backup failed` message and `http_code` variable will contain it. As a result, if statement below will fail
Also we can use `-s` option instead of `2>/dev/null`

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/77/commits/ded89cf369cff7751497a0217f706907a33090cb